kuka_rsi_hw_interface/src/rsi_control_node.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Áron Svastits
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <thread>
+#include <memory>
+
+#include "controller_manager/controller_manager.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/bool.hpp"
+
+
+int main(int argc, char ** argv)
+{
+  rclcpp::init(argc, argv);
+  auto executor = std::make_shared<rclcpp::executors::MultiThreadedExecutor>();
+  auto controller_manager = std::make_shared<controller_manager::ControllerManager>(
+    executor,
+    "controller_manager");
```

Comment:
I think in this case we could use the same controller manager for multiple drivers (maybe with a little extension to have a counter about currently not-unconfigured hw interfaces)
Robot managers will have to be unique, will fix that

Reply:
or is it possible to use the same controller manager for multiple hardware with blocking reads? if not, than it might also be necessary to rename the control nodes, however you can always add a namespace (or I think also change the node name) in a launch file

Reply:
On the robot manager name: to have an identical API, which is high-prio, the nodes should have the same name by default, so I will roll-back the name changes
If more robots are needed, one can add a namespace in the launch file -> maybe all launch files should be extended with a namespace param

Reply:
You could use the same manager for multiple blocking hws when async components get merged. Otherwise they may have a timeout, such as the rox driver. 

Here I was talking about the abstracted lifecycle nodes having the same name, which could lead to unusual behavior.

Reply:
I understand the issue and it seems that remapping the names in the launch files does not solve this. But as this issue was already present before this PR for the other two drivers, I would try to solve this in a different PR, especially as this scenario is not likely for now. Is that okay for you @VX792?

Reply:
It's fine of course